modules/eco-setting-node-maintenance-actions-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-actions-web-console_{context}"]
+= Setting a bare-metal node to maintenance mode
+Set a bare-metal node to maintenance mode using the Options menu {kebab} found on each node in the *Compute* -> *Nodes* list, or using the *Actions* control of the *Node Details* screen.
+
+[NOTE]
```

Comment:
This NOTE is no longer needed, since this NOTE is already under `Working with bare-metal nodes`. Please delete this NOTE.

Reply:
Removed as requested http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#eco-setting-node-maintenance-actions-web-console_node-maintenance-operator